hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -1933,9 +1935,9 @@ public void run() {
                     return;
                 }
                 if (logger.isFinestEnabled()) {
-                    logger.finest("RepairPartitionTableTask scheduled");
+                    logger.finest("FixPartitionTableTask scheduled");
```

Comment:
The related docs always use `repair` word while describing this process. I'm in favor of reverting this repair-> fix change.

Reply:
reverted, having redopartitioning should be sufficient.